app/js/controller/list_controller.js

```diff
@@ -137,9 +145,22 @@ export default class ListController extends Controller {
     }
 
     installReq.onerror = (err) => {
-      this.appView.showAlertDialog('Error installing: ' + err.target.error.name);
-      console.log('install error', err);
+      var errorType = err.target.error.name;
+      switch(errorType) {
+        case 'DENIED':
+          // If the user cancelled the install, we do nothing.
+          break;
+
+        case 'NETWORK_ERROR':
+          this.showAlertDialog('Install Error: No network');
+          break;
+
+        default:
+          this.showAlertDialog('Error installing: ' + err.target.error.name);
+          break;
+      }
```

Comment:
I'm really nitpicking, but it would be better to structure this like so:
```js
var errorMsg;
switch (errorType) {
case 'DENIED':
  // If the user cancelled the install, we do nothing.
  break;
case 'NETWORK_ERROR':
  errorMsg = 'Install Error: No network';
  break;
default:
  errorMsg = 'Error installing: ' + err.target.error.name;
  break;
}

if (errorMsg) {
  this.showAlertDialog(errorMsg);
}
```